app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -152,6 +152,9 @@ AccountService = ($state, $auth, $modal, $http, $translate, ShortFormApplication
         contact: Service.userDataForSalesforce()
       $http.put('/api/v1/account/update', params).success((data) ->
         Service.accountSuccess.messages.nameDOB = $translate.instant("ACCOUNT_SETTINGS.ACCOUNT_CHANGES_SAVED")
+        data.contact.signedIn = true
```

Comment:
@chrisdolendo good question -- the point was to copy over the contact into the loggedInUser while also preserving `signedIn = true` however I'm realizing this can be done better w/ a merge instead of a copy. Will update that, good catch!